public/js/util/sources-tree.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-const URL = require("url").parse;
+const URL = require("url");
```

Comment:
Yep, webpack just bundles this library and uses it instead. I explicit to `URL.parse` to be clear that we aren't using the native one. This is important because they have different semantics: the native URL will throw for invalid URLs like `/foo/bar.js` (it's a path, not a URL, URLs need the protocol). But this library will not throw, it will return `{ protocol: null, pathname: "/foo/bar.js" }`.

I don't like that the library is not compliant with browsers, but if we need this to work in node, we need to depend on it.